navigation.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/navigation.css" />
+    <script
+      src="https://kit.fontawesome.com/43943f2889.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Document</title>
+  </head>
+  <body>
+    <nav class="navigation">
+      <div class="nav-logo">SANGUINE</div>
+      <ul class="list-style-none list-inline">
+        <li>
+          <i class="fa-solid fa-layer-group"></i>
+          <a class="options link" href="/navigation.html">components</a>
+        </li>
+        <li>
+          <label for="drop-down" class="options"></label>
+          <select name="pages" id="drop-down">
+            <option value="default">--Select--</option>
+            <option value="alerts">Alert</option>
+            <option value="buttons">Button</option>
+            <option value="avatar">Avatar</option>
+            <option value="badges">Badge</option>
+            <option value="badges">Image</option>
+            <option value="badges">Input</option>
+            <option value="badges">Typography</option>
+            <option value="badges">List</option>
+            <option value="badges">Navigation</option>
+          </select>
+        </li>
+        <li>
+          <i class="fa-solid fa-house"></i>
+          <a class="options link link-active" href="/navigation.html">home</a>
+        </li>
+        <li>
+          <i class="fa-brands fa-github"></i>
+          <a class="options link" href="/navigation.html">github</a>
```

Comment:
You should add target="_blank", so that github link opens in new tab and you Component Library remains open in current one.

Reply:
Have updated the code. Please approve PR.